src/utils.js

```diff
@@ -40,6 +44,34 @@ export const parseWhere = request => {
   return where;
 };
 
+export const parseLimitAndOffset = (request) => {
+  const { limit, offset } = request.query;
+  return {
+    limit: isUndefined(limit) ? void 0 : toNumber(limit),
```

Comment:
It's a (bad?) habit of mine from the bad old days of JS libs being of poor quality and accidentally (or intentionally!) overriding `undefined` with something else. Turns out, it's a micro-performance optimization to use `void 0` and it _is_ safer.

I also find that writing `undefined` is usually an anti-pattern (usually, you want `null` to express that the value is unset).